app/models/storer.rb

```diff
@@ -34,17 +33,10 @@ def update_device(parsed_ts, sql_data)
     @device.update_columns(last_recorded_at: parsed_ts, data: sql_data, state: 'has_published')
   end
 
-  def redis_publish(readings, ts, stored)
-    return unless Rails.env.production? and @device
+  def redis_publish()
+    return unless !Rails.env.test? and @device
     begin
-      Redis.current.publish("data-received", {
-        device_id: @device.id,
-        device: JSON.parse(@device.to_json(only: [:id, :name, :location])),
-        timestamp: ts,
-        readings: readings,
-        stored: stored,
-        data: JSON.parse(ActionController::Base.new.view_context.render( partial: "v0/devices/device", locals: {device: @device, current_user: nil}))
-      }.to_json)
+      Redis.current.publish("data-received", ActionController::Base.new.view_context.render( partial: "v0/devices/device", locals: {device: @device, current_user: nil}))
```

Comment:
I guess this will change the final object and make it simpler, and that the consumer is also ready for this change?

Reply:
Exactly, consumer has been updated on documentation and the one on production will be updated when changes are rolled